site/docs/hosting/zeabur-deno.md

```diff
@@ -0,0 +1,87 @@
+---
+prev: false
+next: false
+---
+
+# Hosting: Zeabur (Deno)
+
+[Zeabur](https://zeabur.com) is a platform that allows you to deploy your full-stack applications with ease. It supports various programming languages and frameworks, including Deno and grammY.
+
+This tutorial will guide you how to deploy your grammY bots with Deno to [Zeabur](https://zeabur.com/).
+
+::: tip Looking for the Node.js Version?
+This tutorial explains how to deploy a Telegram bot to Zeabur using Deno.
+If you're looking for the Node.js version, please check out [this tutorial](./zeabur-nodejs) instead.
+:::
+
+## Prerequisites
+
+To follow along, you need to have a [Github](https://github.com) and [Zeabur](https://zeabur.com/) account.
+
+### Method 1: Create a New Project From Scratch
+
+> Make sure you have Deno installed on your local machine. 
+
+Initialize your project and install some necessary dependencies:
+
+```sh
+# Initialize the project.
+mkdir grammy-bot
+cd grammy-bot
+
+# Create main.ts file
+touch main.ts
+
+# Create deno.json file to generate lock file
+touch deno.json
+```
+
+Then modify `main.ts` file with the following code:
+
+```typescript
+import { Bot } from "https://deno.land/x/grammy@v1.23.0/mod.ts";
+
+const bot = new Bot("your_telegram_bot_token"); 
```

Comment:
The Node version reads the token from the environment. Is there a reason why this isn't done here?